test/parallel/test-http-agent-maxsockets.js

```diff
@@ -0,0 +1,55 @@
+var common = require('../common');
+var assert = require('assert');
+var http = require('http');
+var Agent = require('_http_agent').Agent;
```

Comment:
This was recently changed and should be `var Agent = require('http').Agent` now.